airtable-automations/SyncRegistrationData.js

```diff
@@ -0,0 +1,205 @@
+// SyncEventAttendees.js
```

Comment:
Can you update this to match the actual file name? Had this in here so that we know what file to look for from airtable